common/system.h

```diff
@@ -233,11 +246,13 @@ class OSystem : Common::NonCopyable {
 	 * implement clipboard support.
 	 */
 	Common::U32String _clipboard;
+	/**
 
-	// WORKAROUND. The 014bef9eab9fb409cfb3ec66830e033e4aaa29a9 triggered a bug
-	// in the osx_intel toolchain. Adding this variable fixes it.
+	/* Workaround for a bug in the osx_intel toolchain introduced by
```

Comment:
Looks like there is a comment in a comment here. Some compilers will complain about that.

Reply:
corrected, thanks